.github/workflows/ghstack_land.yml

```diff
@@ -0,0 +1,28 @@
+name: Propose to merge ghstack orig PRs to main
+on:
+  pull_request:
+    types: [opened, synchronize, closed]
+    branches:
+      - 'gh/*/[0-9]+/base'
```

Comment:
I think if you want to choose a smaller group of people to test this out first, you can list them here explicitly, i.e.

```
    branches:
      - 'gh/kirklandsign/[0-9]+/base'
      - 'gh/guangy10/[0-9]+/base'
```

It seems like a good idea to try it out first before rolling out to everyone.